src/accel/cuda/GPUSolver.h

```diff
@@ -14,22 +14,33 @@
 #include "../../Solver.h"
 #endif
 
-#define PySys_WriteStdout printf
-
-#include <thrust/reduce.h>
 #include <thrust/device_vector.h>
+#include <thrust/copy.h>
+#include <thrust/fill.h>
+#include <thrust/reduce.h>
+#include <thrust/replace.h>
+#include <thrust/functional.h>
+#include <thrust/iterator/constant_iterator.h>
 #include <sm_20_atomic_functions.h>
 #include "clone.h"
 #include "GPUExpEvaluator.h"
 
+#define PySys_WriteStdout printf
```

Comment:
See if my latest commit works for CUDA 6 and 7, it works for me with 5.5 and 6.5.